### DISCUSSION TITLE
How to export filamat or mat in running program After i dynamic call setParameter on MaterialInstance

### DISCUSSION THREAD

**Discussion by dandingol03** [google/filament#4640] [Q&A] [answered]
How to export filamat or mat in running program After i dynamic call setParameter on MaterialInstance. i can get the material, but it seems like there is no property about the ``fragment`` statement or ``vertex`` statement

**Comment by romainguy** [comment, marked as answer]
You cannot export a material at runtime. Note that material files do not contain parameter values.
